### PR TITLE
Configure UPS server with mock URLs for FCM

### PR DESCRIPTION
**Pull request by josemigallas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,5 +15,10 @@ To start it again use:
 ```
 java -jar path/to/server/wiremock-standalone-2.4.1.jar --port 3000
 ```
+Once it is running, start your UPS setting `-Dcustom.aerogear.fcm.push.host` parameter with the facked URL:
```
Comment (ziccardi) [inline README.md]: Probably you mean `fake URL`

File: README.md [hunk 2]
```diff
@@ -15,5 +15,10 @@ To start it again use:
...
+```
+path/to/jboss/bin/standalone.sh -b 0.0.0.0 --server-config=standalone-full.xml Dcustom.aerogear.fcm.push.host=http://localhost:3000/fcm/send
```
Comment (ziccardi) [inline README.md]: Here it should be `-Dcustom.aerogear.fcm.push.host=http://localhost:3000/fcm/send`

**Review by ziccardi — CHANGES_REQUESTED** [review]
Added a couple of comments

**Comment by matzew** [comment]
cool

**Pull request merged**